src/components/Home/HomeContent.tsx

```diff
@@ -0,0 +1,185 @@
+import React from "react";
+import CTARegistration from "./CTARegistration";
+import { Row, Col } from "antd";
+import PersonalityCard from "../Personality/PersonalityCard";
+import SocialMediaShare from "../SocialMediaShare";
+import { useTranslation } from 'next-i18next';
+import Button, { ButtonType } from '../Button';
+import { ArrowRightOutlined } from "@ant-design/icons";
+import SectionTitle from "../SectionTitle";
+import styled from "styled-components";
+
+const MainContent = styled.div`
```

Comment:
Move it to another component. Maybe that's a good opportunity to use the name pattern @thiagobraga mentioned: The new file should be named `HomeContent.style.tsx`